editor/editor_node.h

```diff
@@ -566,6 +566,8 @@ class EditorNode : public Node {
 	void _scene_tab_closed(int p_tab);
 	void _cancel_close_scene_tab();
 
+	void _popup_save_confirmation();
```

Comment:
```suggestion
	void _prepare_save_confirmation_popup();
```
The current name sounds like it would popup the dialog.